### PR TITLE
OS-99 : Add autocomplete fields for the thesis institute and the thesis location

### PR DESCRIPTION
**Pull request by jcougnaud**
- [X] https://github.com/uclouvain/osis-admission/pull/41 must be merged before.
- [X] A new sdk must be generated and then used in this PR.
- [X] Somes changes are needed in the osis-organisation-sdk to search the institutes with their title / acronym and to retrieve one from its uuid  (cf https://github.com/uclouvain/osis/pull/11288)

### CODE REVIEW

File: contrib/views/autocomplete.py [hunk 1]
```diff
@@ -135,3 +140,52 @@ def get_list(self):
             person=self.request.user.person,
             search=self.request.GET.get('q', ''),
         )
+
+
+class EntityAutocomplete(LoginRequiredMixin, autocomplete.Select2ListView):
```
Comment (SebCorbin) [inline contrib/views/autocomplete.py:145]: InsituteAutocomplete ?

File: contrib/views/autocomplete.py [hunk 2]
```diff
@@ -135,3 +140,52 @@ def get_list(self):
...
+        return results
+
+    def results(self, results):
+        return [dict(
+            id=entity.uuid,
+            text=format_entity_title(entity=entity),
+        ) for entity in results]
+
+
+class EntityPlaceAutocomplete(LoginRequiredMixin, autocomplete.Select2ListView):
```
Comment (SebCorbin) [inline contrib/views/autocomplete.py:168]: InsituteLocationAutocomplete ?

**Review by SebCorbin — APPROVED** [review]

File: services/organisation.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+class EntitiesAPIClient:
+    def __new__(cls):
+        api_config = organisation_sdk.build_configuration()
+        return entites_api.EntitesApi(ApiClient(configuration=api_config))
+
+
+class EntitiesService:
+    @classmethod
+    def get_entities(cls, person, organisation_code, entity_type, *args, **kwargs):
```
Comment (BenJneB) [inline services/organisation.py:41]: On a besoin que des entités UCL dans le contexte d'admission, il me semble.
Je renommerais la méthode et virerais le paramètre organisation_code :

`    def get_ucl_entities(cls, person, entity_type, *args, **kwargs):`

Comment (jcougnaud) [reply services/organisation.py]: D'accord ça marche, j'ai simplifié pour les trois fonctions.

**Review by BenJneB — APPROVED** [review]

**Pull request merged**